README.md

```diff
@@ -303,6 +303,18 @@ This is because the database is unaware of the plain-text data (which is part of
 the security model).
 
 
+### Key rotation
+After rotating the key inside the Vault, all new records will be encrypted with
+new key. If you wish to rotate old records to the new key you need to update
```

Comment:
[rewrap](https://www.vaultproject.io/api/secret/transit/index.html#rewrap-data) is a Vault endpoint that can help with this.